src/main.rs

```diff
@@ -119,9 +126,11 @@ fn main() {
         }
     };
 
+    let policies_download_dir = matches.value_of("policies-download-dir").unwrap();
```

Comment:
Super nit: I wonder what happens if the command gets executed with `--poilcies-download-dir ""`, is anything preventing this from happening?

Reply:
I've updated the code to ensure this doesn't happen.